index.html

```diff
@@ -9,26 +9,183 @@
-	<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
-	  
-	</ul>
-    </section>
+    <div class='mainsection'>
+	    <header class="header">
+	      <a class="mylogo">PRATAP PANABAKA</a>
+	      <nav class='nav'>
+		<img src='images/hamburg.svg' alt="Click for the Menu Options" title=''>
+	      </nav>
+	    </header>
+	    <section>
+	     	<p class="myheader">I’m PRATAP Glad to see you!</p>
+	    	<p class="content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+	    	<h3 class="connect">LET’S CONNECT</h3>
+		<ul class="social-media">
+
+		<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt="Twitter Profile" title=''></a></li>
+		<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt="LinkedIn Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
+		<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt="GitHub Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
+		  
+		</ul>
+	    </section>
+    </div>
+    <div class='projects-section'>
+    <div class="project-wrapper grid-box">
+    
+    	<div class="project">
+    		<img src="projects/project1.png" alt="project 1" title="Project 1" width="295" height="220" />
+    		<div class="project-heading">
+    			<h2 class="project-title">Project 1</h2>
+    			<div class="project-meta-data">
+    				<div class="project-client">Linux</div>
+    				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-tech">Linux User</div>
+				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-year">2015</div>
+			</div>
+			<ul class="project-tags">
+
+				<li class="tags">html</li>
+				<li class="tags">css</li>
+				<li class="tags">javaScript</li>
+			
+			</ul>
+			<div class="project-description">
+                    		A daily selection of privately personalized reads; no accounts or sign-ups required.
+                	</div>
+			<div class="project-button">
+                		<button type="button">See Project</button>
```

Comment:
- [ ] Great job implementing button states for normal and on hover :clap: . However, according to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), you might have left out the "On active" state. Kindly make sure you implement this as it will greatly improve your websites UI/ UX.
 - Below I have included what the button states should look like.

**Button states according to Figma design.**
![image](https://user-images.githubusercontent.com/86318284/176887474-955bfcaa-7529-4fef-bdf2-ea018cf14214.png)

**Kindly make sure you implement this on all your buttons**